interwovenkit/references/components/interwovenkit-provider.mdx

```diff
@@ -0,0 +1,180 @@
+| `disableAnalytics`       | `boolean`                             | `false` (mainnet), `true` (testnet) | When `true`, disables InterwovenKit's built‑in analytics events                                                                                                                                                                                                                                                               |
+| `enableAutoSign`         | `boolean \| Record<string, string[]>` | `undefined`                         | Enables AutoSign and optionally whitelists chains and message types. `true` enables for supported chains and messages. `Record<string, string[]>` is a per-chain allowlist of message type URLs. **Requires Privy** (`PrivyProvider` and `privyContext`). Required at runtime only if you want AutoSign flows to be available |
+| `privyContext`           | `PrivyContext`                        | `undefined`                         | Passes Privy authentication and wallet helpers into InterwovenKit. **Required for AutoSign and embedded wallet features**, otherwise optional                                                                                                                                                                                 |
+
+## Return value
+
+Renders `children` and mounts the InterwovenKit UI shell.
+
+## Examples
+
+### Custom chain configuration
+
+```tsx
+import type { Chain } from '@initia/initia-registry-types'
+import { InterwovenKitProvider, MAINNET } from '@initia/interwovenkit-react'
+
+const MY_ROLLUP: Chain = {
+  chain_id: 'my-rollup-1',
+  chain_name: 'my-rollup',
+  pretty_name: 'My Rollup',
+  network_type: 'mainnet',
+  bech32_prefix: 'init',
+  fees: {
+    fee_tokens: [
+      {
+        denom: 'uinit',
+        fixed_min_gas_price: 0.1,
+      },
+    ],
+  },
+  apis: {
+    rpc: [{ address: 'https://rpc.my-rollup.com' }],
+    rest: [{ address: 'https://api.my-rollup.com' }],
+    indexer: [{ address: 'https://indexer.my-rollup.com' }],
+  },
+} as Chain
+
+export function Providers({ children }: { children: React.ReactNode }) {
+  return (
+    <InterwovenKitProvider
+      {...MAINNET}
+      defaultChainId={MY_ROLLUP.chain_id}
+      customChain={MY_ROLLUP}
+    >
+      {children}
+    </InterwovenKitProvider>
+  )
+}
+```
+
+<Note>
+  This example shows only the `InterwovenKitProvider` configuration. For
+  complete provider setup including `QueryClientProvider`, `WagmiProvider`, and
+  `injectStyles`, see [Provider Setup](../setup/providers).
+</Note>
+
+## Notes
+
+- For AutoSign and Privy integration setup, see
+  [Provider Setup](../setup/providers).
```

Comment:
Better to use <Note> component and place it at the top of the page or somewhere in-context